robot/client/client.go

```diff
@@ -363,6 +372,31 @@ func New(ctx context.Context, address string, clientLogger logging.ZapCompatible
 		}
 	}
 
+	// if withNetworkStats is set, there are latencyPingNum gRPC requests sent to the server
+	// to measure the average latency of the connection, which is then logged to the user.
+	if rOpts.withNetworkStats {
+		totalTime := 0.0
+		for range latencyPingNum {
+			startTime := time.Now()
+			_, err := rc.client.GetOperations(ctx, &pb.GetOperationsRequest{})
+			if err != nil {
+				rc.Logger().CDebug(ctx, fmt.Sprintf("gRPC request failed with error: %e during latency checks", err))
+				continue
+			}
+			timeElapsed := time.Since(startTime).Milliseconds()
+			totalTime += float64(timeElapsed)
+		}
+		if totalTime < 1.0 {
```

Comment:
This conditional looks incorrect to me; do we want to be looking at `totalTime` here?